### PR TITLE
PR after group mates comments

### PR DESCRIPTION
**Pull request by andrewryh**
This PR resolves issues #21 #20 #19 #22 #23

### CODE REVIEW

File: results.tex [hunk 1]
```diff
@@ -20,66 +20,65 @@ \section{Results}
...
 the fuel salt change insignificantly after approximately 2500 days of operation. 
-Particularly, the $^{233}$U number density fluctuates less than 0.8\% in the time 
-interval from 16 to 20 years of operation. Hence, a quasi-equilibrium state was 
+In particular, the $^{233}$U number density fluctuates by less than 0.8\% between
+ 16 and 20 years of operation. Hence, a quasi-equilibrium state was 
 achieved after 16 years of reactor operation.
 \begin{figure}[ht!] % replace 't' with 'b' to 
   \centering
-  \includegraphics[width=\textwidth]{major_isotopes_adens.png}
+  \includegraphics[scale=•]{•}=\textwidth]{major_isotopes_adens.png}
```
Comment (katyhuff) [inline results.tex]: This change causes a build error for me.  Seems like some kind of typo

**Comment by andrewryh** [comment]
OMG, that is the shame.
Broke it occasionally when switched between windows using Alt+Tab

File: methods.tex [hunk 2]
```diff
@@ -313,14 +308,14 @@ \subsubsection{The SaltProc modeling and simulation code}
...
-multiple depletion materials. At the end of a each depletion step, SaltProc 
+multiple depletion materials. At the end of each depletion step, SaltProc 
 reads the depleted compositions and tracks each material stream individually. 
 Following this, it applies chemical separation functions to fuel stream 
 vectors. These vectors then form a matrix (isotopics x timesteps) which 
 SaltProc stores in an HDF5 database and prints into the SERPENT 2 composition 
 file for the next depletion calculation.
 
-Saltproc datasets are timseries, meaning that every value is recorded every 
+Saltproc datasets are time-series, meaning that every value is recorded every
```
Comment (katyhuff) [inline methods.tex]: I don't think I like hyphenating time series. Perhaps "SaltProc records every value every timestep. The resulting time series datasets produced by SaltProc are listed below.... "

File: results.tex [hunk 3]
```diff
@@ -71,15 +70,15 @@ \subsection{Fuel salt composition dynamics}
   \label{fig:adens_eq}
 \end{figure}
 In contrast, a wide variety of nuclides, including fissile isotopes (e.g. 
-$^{235}$U) and non-fissile strong absorbers (e.g. $^{234}$U), keep accumulating 
+$^{235}$U) and non-fissile strong absorbers (e.g. $^{234}$U), kept accumulating 
 in the core. Figure~\ref{fig:fissile_short} demonstrates production of fissile 
 isotopes in the core. In the end of the considered operational time, the core 
-contains significant $^{235}$U ($\approx10^{-5}$ atom/b-cm), $^{239}$Pu 
+contained significant amount of $^{235}$U ($\approx10^{-5}$ atom/b-cm), $^{239}$Pu
```
Comment (katyhuff) [inline results.tex]: delete "amount of"

File: results.tex [hunk 4]
```diff
@@ -131,24 +130,24 @@ \subsection{Neutron spectrum}
...
-neutron flux for both initial and equilibrium composition. The neutron flux has 
-the same shape for both compositions but the equilibrium case has a harder 
-spectrum. A significant spectral shift was observed for the central region of 
-the core (zone I) when for the outer region (zone II) it is negligible for fast 
-but notable for thermal neutrons. This neutron flux radial distribution is in a 
+neutron flux for the both initial and equilibrium composition. The neutron fluxes
+have similar shapes for both compositions but the equilibrium case has a harder 
+spectrum. A significant spectral shift was observed in the central region of 
+the core (zone I), while for the outer region (zone II), it is negligible for fast 
+but notable for thermal neutrons. These neutron flux radial distributions is in
```
Comment (katyhuff) [inline results.tex]: "These... distributions... are"

File: results.tex [hunk 5]
```diff
@@ -170,11 +169,12 @@ \subsection{Power and breeding distribution}
...
 and equilibrium states. The distribution reflects the spatial distribution of 
 $^{233}$Th production in the core. The thorium-232 then $\beta$-decays to 
-$^{233}$Pa which is the precursor for $^{233}$U production. Accordingly, this 
+$^{233}$Pa, which is the precursor for $^{233}$U production. Accordingly, this 
 characteristic represents the breeding distribution in the \gls{MSBR} core. 
 Spectral shift does not cause significant changes in power nor in breeding 
-distribution. Even after 20 years of operation, most of the power still is 
-generated in zone I though the majority of $^{233}$Th is produced in zone II.
+distribution. Even after 20 years of operation, most of the power is still 
+generated in zone I with exception to the majority of $^{233}$Th that is
```
Comment (katyhuff) [inline results.tex]: with exception to --> except

File: results.tex [hunk 6]
```diff
@@ -274,7 +274,7 @@ \subsection{Reactivity control system rod worth}
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 
 \subsection{Six Factor Analysis}
-The effective multiplication factor could be expressed using formula:
+The effective multiplication factor can be expressed using next formula:
```
Comment (katyhuff) [inline results.tex]: "using the following formula:"

**Review by katyhuff — CHANGES_REQUESTED** [review]
minor comments here.  looking great! (I have a few more edits, but I'll submit those changes myself as a pr)

File: methods.tex (hunk 2 above)
Comment (katyhuff) [reply methods.tex]: see https://github.com/andrewryh/msbr-reproc/pull/3

File: results.tex (hunk 3 above)
Comment (katyhuff) [reply results.tex]: https://github.com/andrewryh/msbr-reproc/pull/3

File: results.tex (hunk 4 above)
Comment (katyhuff) [reply results.tex]: https://github.com/andrewryh/msbr-reproc/pull/3

File: results.tex (hunk 5 above)
Comment (katyhuff) [reply results.tex]: see https://github.com/andrewryh/msbr-reproc/pull/3

File: results.tex (hunk 6 above)
Comment (katyhuff) [reply results.tex]: see https://github.com/andrewryh/msbr-reproc/pull/3

**Comment by katyhuff** [comment]
Once see https://github.com/andrewryh/msbr-reproc/pull/3 is merged into this PR, all of my comments will have been handled and I will change review to 'approve'

File: introduction.tex [hunk 7]
```diff
@@ -21,14 +21,14 @@ \section{Introduction}
 safety parameters, reduced fuel preprocessing, and the ability to continuously 
 remove fission products and add fissile and/or fertile elements 
 \cite{leblanc_molten_2010}. The thorium-fueled \gls{MSBR} was developed in the 
-early 1970s by \gls{ORNL} specifically to explore the promise of the thorium 
+early 1970s by \gls{ORNL} to explore specifically the promise of the thorium 
 fuel cycle, which uses natural thorium instead of enriched uranium. With 
 continuous fuel reprocessing, the \gls{MSBR} realizes the advantages of the 
 thorium fuel cycle because the $^{233}$U bred from 
-$^{232}$Th is almost instantly \footnote{\space $^{232}$Th transmutes into 
+$^{232}$Th is almost instantly\footnote{\space $^{232}$Th transmutes into
```
Comment (anshchaube) [inline introduction.tex]: $^{232}$Th is almost instantly\footnote{\space $^{232}$Th transmuted into

**Review by anshchaube — CHANGES_REQUESTED** [review]

**Comment by katyhuff** [comment]
@andrewryh  -- please don't add more to this PR. New issues can be handled in separate PRs. Please complete this PR approval by merging  https://github.com/andrewryh/msbr-reproc/pull/3 . Then, we can merge this PR and it will be safe to move on to other issues.

**Pull request merged**